Rules/mtbmap.mrules

```diff
@@ -1844,9 +1829,10 @@ rules
 				min-zoom : 12
 				fill-color :#f4e5d3
 				painting-cycle : 401
-			draw : fill
+
 				
-		
+		draw : text
+		draw : fill
```

Comment:
having `draw : fill` and `draw : text` together does not allow having different zoom ranges for fill and text.

I suggest the `draw : text` will be added to the relevant areas only instead of here. It should be proceeded with the zoom settings for the text and followed by the zoom settings for the the fill.

It will also enable replacing the initial `text : ""` in line 1715 with `text-func : NodeName(e)` and eliminating the need for additional `text-func` definitions in `area *`